src/explorer/src/components/Endpoint/Geocoding/StreetZone/StreetZoneApi.js

```diff
@@ -34,21 +34,17 @@ export default function StreetZone(props) {
 
   return (
     <>
-      <EndpointInput name="street" {...defaultAttributes.street} dispatch={dispatch} />
-      <EndpointInput name="zone" {...defaultAttributes.zone} dispatch={dispatch} />
+      <EndpointInput name="street" schema={schema.fields.street.clone()} dispatch={dispatch} />
```

Comment:
**question**: did you learn that `clone` is required? What happens if it isn't there?